### PR TITLE
OP-27: craniovertebral_angle_deg — forward head posture

### PR DESCRIPTION
**Pull request by m-nweke**
This pull request introduces a new metrics subsystem for posture analysis, with a focus on clear, robust, and testable measurement of key clinical angles. It adds a modular architecture for metrics, implements two core metrics (`craniovertebral_angle_deg` and `trunk_inclination_deg`), and provides comprehensive test scaffolding and tests to ensure correctness and reliability.

**New Metrics Subsystem**

- Added a new metrics package structure with a unified interface for all metrics, ensuring each metric is isolated in its own module and returns a standardized `Metric` result. (`packages/posture-core/src/posture_core/metrics/__init__.py`)

- Introduced `_support.py` with shared utilities for metric modules, including helpers for abstaining from a measurement, fetching world coordinates, and safely running computations with explicit error handling. (`packages/posture-core/src/posture_core/metrics/_support.py`)

**Implemented Metrics**

- Added `craniovertebral_angle_deg` in `head.py`, measuring forward head posture using a clinically accurate method, with robust handling for missing data and configurable thresholds. (`packages/posture-core/src/posture_core/metrics/head.py`)

- Added `trunk_inclination_deg` in `trunk.py`, measuring trunk lean with improvements over the legacy implementation, including correct sign convention, world-space measurement, and explicit error reporting. (`packages/posture-core/src/posture_core/metrics/trunk.py`)

**Test Infrastructure and Coverage**

- Added `builders.py` to provide test scaffolding for constructing synthetic postures and running metrics in a controlled, analytic environment, enabling precise and meaningful tests. (`packages/posture-core/tests/builders.py`)

- Added comprehensive tests for `craniovertebral_angle_deg`, covering geometric correctness, threshold handling, abstention scenarios, and edge cases. (`packages/posture-core/tests/test_head.py`)

### CODE REVIEW

File: packages/posture-core/src/posture_core/metrics/head.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+scores a reduced angle, because the reference is gravity rather than the torso. That is correct:
+forward head posture is defined against the vertical, and the load on the neck depends on where
+the head is in space, not on where it is relative to a tilted spine.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Final
+
+from posture_core.geometry import angle_between, midpoint
```
[the remark this answers is not in the document]
Comment (m-nweke) [reply packages/posture-core/src/posture_core/metrics/head.py:31]: The premise does not hold — the CVA computation was never changed to sagittal-plane inclination, so `angle_between` is still the function being used:

```
 31: from posture_core.geometry import angle_between, midpoint
 90:         compute=lambda: angle_between(ear - c7, forward),
```

`signed_angle_to_vertical` would be the wrong primitive here. The craniovertebral angle is defined against the **horizontal**, unsigned, and values above 90° simply mean the head sits behind the shoulders. `ruff` would also have failed the build on an unused import — `F401` is enabled and `lint` is green.

Leaving as is.
